pyface/util/image_helpers.py

```diff
@@ -0,0 +1,31 @@
+# (C) Copyright 2005-2021 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+""" Helper functions for working with images
+
+This module provides helper functions for converting between numpy arrays
+and toolkit images, as well as between the various image types in a standardized
+way.
+"""
```

Comment:
```suggestion
This module provides helper functions for converting between numpy arrays
and toolkit images, as well as between the various image types in a standardized
way.

Helper functions
----------------

- :data:`~.array_to_image`
- :data:`~.bitmap_to_icon`
- :data:`~.bitmap_to_image`
- :data:`~.image_to_array`
- :data:`~.image_to_bitmap`
- :data:`~.resize_image`
- :data:`~.resize_bitmap`

Options for resizing images
---------------------------
- :data:`~.ScaleMode`
- :data:`~.AspectRatio`

"""
```

Reply:
I'm assuming this is needed because of the way that API docs are generated?

Reply:
yep. i thought it'll make the autogenerated api docs better.